### ISSUE TITLE
libdivsufsort should use int_vector<> 

### ISSUE THREAD

**Issue by mpetri** [simongog/sdsl#30]
now that libdivsufsort is included in the library it would be very nice to modify it to use the sdsl int vector to enable bit compression among other things.

**Comment by simongog** [comment]
Yes, we can modify that. I will also extend the memory management system in such a way that single int_vectors can be mapped to hugepages and we therefore gain time in the construction process.
